src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs

```diff
@@ -18,13 +19,12 @@ public class Startup : FunctionsStartup
         /// <param name="builder">The instance to build the registered services inside the functions app.</param>
         public override void Configure(IFunctionsHostBuilder builder)
         {
-            IConfiguration configuration =
-                new ConfigurationBuilder()
+            IConfiguration configuration = new ConfigurationBuilder()
                     .SetBasePath(builder.GetContext().ApplicationRootPath)
                     .AddEnvironmentVariables("ARCUS_")
-                    .AddJsonFile("local.settings.json")
+                    .AddJsonFile("local.settings.json", optional: false)
```

Comment:
Time wasted : 9000

Reply:
Uh? That's so weird. Maybe also the namespace of the startup file.

Reply:
The file is not there so it blew up, but didn't give an error..